### DISCUSSION TITLE
Separate navigation from trip logging

### DISCUSSION THREAD

**Discussion by sweeoon** [osmandapp/OsmAnd#17719] [Ideas]
Why is it that when I stopped navigating a track half way, Osmand also stopped trip logging? There was no warning or alert. And the logged trip up to that point was not saved. I lost that logged trip.

Please separate navigating a track from trip logging. When I start navigating a saved gpx file, I also have to start trip logging separately. BUT when I stopped navigating the track before it's end, Osmand also terminates the trip logging without giving me a choice to save the trip. I should be able to stop navigating a track at any time without Osmand also terminating trip recording.

**Comment by DmitryAlexei** [comment]
Please add info:
- OsmAnd version (Menu > Help > About (long tap to copy)) 
- Android version
- about device you use

**Reply by sweeoon** [reply]
Version: OSMAnd+ 4.5.9, released: 2023-08-01
Android version: 11
Doogee V10 smartphone, built 17 Feb 2022.

**Reply by DmitryAlexei** [reply]
Has trip recording stopped in background? If so, try to disable Battery optimization for OsmAnd in your system settings.
Possible way to this setting is: Android System settings > Battery > Battery usage (or Battery > Power saving)

**Reply by DmitryAlexei** [reply]
Please note
After stopping navigation, OsmAnd returns to the base profile (it may be different from the one used for navigation and may not have a trip recording widget). 
If it is your case, please switch app back to the navigation profile and check Trip recording widget.
